applications/applications_interface.py

```diff
@@ -135,6 +138,15 @@ def ImportApplications(kernel, applications_path=application_directory):
     print("Import_ConstitutiveModelsApplication: " + str(Import_ConstitutiveModelsApplication))
     print("Import_ShallowWaterApplication: " + str(Import_ShallowWaterApplication))
 
+    if(Import_FluidRveLagrangeMultipliersApplication):
+        print("importing KratosFluidRveLagrangeMultipliersApplication ...")
+        sys.path.append(applications_path + '/fluid_rve_lagrange_multipliers_application/python_scripts')
```

Comment:
Naming conventions for applications is UpperCamleCase

Btw I would recommend to make a separate Pull request for adding a new application, this title of this PR is misleading

Reply:
I guess I would create a branch from this one and revert all unwanted changes such that the new Branch contains only the `FluidRveLagrangeMultipliersApplication`

Not the expert in git though, there might be better options

Reply:
greetings dear,

 the fluid_rve application was removed from the branch by Dr. Pavel,

best regards,